src/scaffoldmaker/annotation/body_terms.py

```diff
@@ -8,6 +8,8 @@
     ( "thorax", "ILX:0742178" ),
```

Comment:
I looked up thorax and abdomen in SciCrunch and these include the the outer tissue and skin layers so are not appropriate terms. I'm thinking we need to change them to be:
'thoracic cavity'
'abdominal cavity'/'pelvic cavity' or combined 'abdominopelvic cavity'.
If so, we need to make the spinal cord slightly outside.
Alternatively we must define our own terms representing what we mean by the core.